tests/utils/test_logging.py

```diff
@@ -12,26 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import unittest
-
+import logging
```

Comment:
Can you mirror the the logging setup? Ie. The test file should mirror the file you are Testing against.